pkg/client/metrics/metrics.go

```diff
@@ -23,25 +23,27 @@ import (
 
 	"github.com/prometheus/client_golang/prometheus"
 
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
 	"sigs.k8s.io/prometheus-adapter/pkg/client"
 )
 
 var (
 	// queryLatency is the total latency of any query going through the
 	// various endpoints (query, range-query, series).  It includes some deserialization
 	// overhead and HTTP overhead.
-	queryLatency = prometheus.NewHistogramVec(
-		prometheus.HistogramOpts{
+	queryLatency = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
 			Name:    "cmgateway_prometheus_query_latency_seconds",
 			Help:    "Prometheus client query latency in seconds.  Broken down by target prometheus endpoint and target server",
-			Buckets: prometheus.ExponentialBuckets(0.0001, 2, 10),
+			Buckets: prometheus.ExponentialBuckets(0.001, 2, 10),
```

Comment:
I think we could use the default buckets: https://github.com/prometheus/client_golang/blob/51d24f86807c24ce866355d0c82258420f571c17/prometheus/histogram.go#L264